src/ophys_etl/transforms/convert_rois.py

```diff
@@ -0,0 +1,194 @@
+                                             "left direction"))
+    mask_image_plane = Int(required=True,
+                           description=("The old segmentation pipeline stored "
+                                        "overlapping ROIs on separate image "
+                                        "planes. For compatibility purposes, "
+                                        "this field must be kept, but will "
+                                        "always be set to zero for the new "
+                                        "updated pipeline"))
+    exclusion_labels = List(Int, required=True,
+                            description=("LIMS IDs used to track why a given "
+                                         "ROI is not considered a valid_roi"))
+
+
+class BinarizeAndCreateROIsOutputSchema(ArgSchema):
+    old_rois = Nested(OldSegmentationROISchema, many=True)
+
+
+class BinarizerAndROICreator(ArgSchemaParser):
+    default_schema = BinarizeAndCreateROIsInputSchema
+
+    def binarize_and_create(self):
+
+        self.logger.name = type(self).__name__
+        self.logger.setLevel(self.args.pop('log_level'))
+
+        # load in the rois from the stat file and movie path for shape
+        self.logger.info("Loading suite2p ROIs and video size.")
+        suite2p_stats = np.load(self.args['suite2p_stat_path'],
+                                allow_pickle=True)
+        with h5py.File(self.args['motion_corrected_video'], 'r') as open_vid:
+            movie_shape = open_vid['data'][0].shape
+
+        # binarize the masks
+        self.logger.info("Binarizing the ROIs created by Suite2p.")
+        coo_rois = suite2p_rois_to_coo(suite2p_stats, movie_shape)
+
+        binarized_coo_rois = []
+        for coo_roi in coo_rois:
+            binary_mask = binarize_roi_mask(coo_roi,
+                                            self.args['abs_threshold'],
+                                            self.args['binary_quantile'])
+            binarized_coo_rois.append(binary_mask)
+        self.logger.info("Binarized ROIs from Suite2p, total binarized: "
+                         f"{len(binarized_coo_rois)}")
+
+        # load the motion correction values
+        self.logger.info("Loading motion correction border values from "
+                         f" {self.args['motion_correction_values']}")
+        motion_correction_df = pd.read_csv(
+            self.args['motion_correction_values'])
+        motion_border = get_max_correction_values(
+            motion_correction_df, self.args['maximum_motion_shift'])
+
+        # create the rois
+        self.logger.info("Transforming ROIs to old segmentation style.")
+        old_segmentation_rois = coo_rois_to_old(binarized_coo_rois,
+                                                motion_border,
+                                                movie_shape)
+
+        # save the rois as a json file to output directory
+        self.logger.info("Writing old style ROIs to json file at "
```

Comment:
Remove "old style" and replace with better descriptor

Reply:
replaced with 'lims_compatible'